helpers/zenhub.py

```diff
@@ -24,11 +33,36 @@ def api(token):
     )
 
 
+def fetch_workspace_repos(op, workspace_id):
+    repos = op.workspace(id=workspace_id).repositories()
+    repos.id()
+    repos.gh_id()
+    repos.name()
+    repos.owner.login()
+
+
+def get_workspace_repos(endpoint, workspaces):
+    repos = []
+
+    for workspace_id in workspaces:
+        op = Operation(zenhub_schema.Query)
+        fetch_workspace_repos(op, workspace_id)
+
+        d = endpoint(op)
+        data = op + d
+
+        if hasattr(data.workspace, 'repositories'):
+            repos += [
+                ((repo.owner.login, repo.name), repo.gh_id, repo.id)
+                for repo in data.workspace.repositories
+            ]
+
+    return repos
+
+
 def fetch_workspace_graph(op, workspace_id, repos, cursor):
     dependencies = op.workspace(id=workspace_id).issue_dependencies(
-        # TODO: This causes a 500 Internal Server Error. We need the ZenHub repo IDs here,
-        # not the GitHub repo IDs (which the previous REST API used).
-        # repository_ids=repos,
+        repository_ids=[repo.zh_id for repo in repos],
```

Comment:
I wouldn't have needed to do any of this if ZenHub supported `repositoryGhIds` here like it does elsewhere. But `repositoryIds` takes ZenHub IDs.